src/store/actions.ts

```diff
@@ -79,5 +79,8 @@ export default {
   },
   setFilingData ({ commit }, filingData: Array<FilingDataIF>) {
     commit('filingData', filingData)
+  },
+  setHasBlockertask ({ commit }, hasBlockerTask: boolean) {
+    commit('hasBlockerTask', hasBlockerTask)
```

Comment:
Search for `has-blocker-task`. This global may conflict with that.

Reply:
indeed! That value is emit from the TODO list up to the dashboard, where it was used locally. I set it to store opposed to a local property, so we can access it in the Entity Info as well, which was mounted at the `app` level.

Reply:
So they conflict or they play well together? Can the event be replaced by the store property instead?

Reply:
No conflicts, the emit value is from child to parent, where it is then set to store. 
I'll have a look if there is any refactoring, but i think it's preferable keeping the child components isolated of the store (as best we can :)).

Reply:
Well, ok, but it feels like a code smell. Why do it 2 different ways -- can't the child just set the store value? Also, if it's only between a child and parent, it may be cleaner to use an `@event` to pass up the event instead of a global emitter. I think global emitters should be reserved to "whole app" events, whereas the parent-child interface (props/events) should be used otherwise.

But if you've already thought about this and think it's the correct way forward, go for it and ignore this comment :)

Reply:
`this.$emit` is child/parent interface no?
 Isn't `this.$root.$emit` global? If so, that is not what is used here.

Reply:
There is nothing wrong with a child setting the store value, but the way we've been evolving is trying to isolate the children to make them more reusable (thinking shared components here). Obviously this component is far beyond that point, but it doesn't hurt to try and follow that practice.

Reply:
1. Oops, I thought this was a global emitter. The old syntax confused me. You're right.
2. Agreed, keep the parent-child API as clean as possible.
3. I'm resolving this conversation. Thanks for humouring me :)

Reply:
No prob! It's always worth the conversation :)